docs/source/markdown/podman-login.1.md

```diff
@@ -111,6 +111,14 @@ $ echo $testpassword | podman login -u testuser --password-stdin docker.io
 Login Succeeded!
 ```
 
+```
+$ podman login quay.io --verbose
+Username: myusername
+Password:
+Modified: /run/user/1000/containers/auth.json
```

Comment:
Doesn't this like show up as

credential helper: /run/user/1000/containers/auth.json

Reply:
Here is what I got.

Used:  /run/user/3267/containers/auth.json

Reply:
@rhatdan Fixed.